### PR TITLE
fix(rust): typo in lsp settings

### PR DESCRIPTION
**Pull request by chaozwn**

### CODE REVIEW

File: lua/astrocommunity/pack/rust/init.lua [hunk 1]
```diff
@@ -9,7 +9,7 @@ return {
       config = {
         rust_analyzer = {
           settings = {
-            ["rust-analyser"] = {
+            ["rust_analyzer"] = {
```
[the remark this answers is not in the document]
Comment (Uzaaft) [reply lua/astrocommunity/pack/rust/init.lua:12]: Actually, I'm pretty sure this is wrong. lsp-config uses the dash.

**Comment by mehalter** [comment]
The original code is correct. Please refer to the rust analyzer docs for reference

**Comment by chaozwn** [comment]
> The original code is correct. Please refer to the rust analyzer docs for reference

sorry, word is incorrectly.

File: lua/astrocommunity/pack/rust/init.lua (hunk 1 above)
Comment (mehalter) [inline lua/astrocommunity/pack/rust/init.lua:12]:
```suggestion
            ["rust-analyzer"] = {
```

I see now the original error. This should be a `-` though

**Review by mehalter — CHANGES_REQUESTED** [review]

**Review by mehalter — APPROVED** [review]
Looks great thanks!

**Pull request merged**